.gitignore

```diff
@@ -1,3 +1,4 @@
 vendor
 .phpunit.result.cache
-composer.lock
\ No newline at end of file
+composer.lock
+.idea
```

Comment:
`.idea` should be added to `.git/info/exclude`

It is not project-related file. So it should not be added to `.gitignore`